ethstorage/scanner/config.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2022-2023, EthStorage.
+// For license information, see https://github.com/ethstorage/es-node/blob/main/LICENSE
+
+package scanner
+
+import (
+	"fmt"
+
+	"github.com/ethstorage/go-ethstorage/ethstorage/rollup"
+	"github.com/urfave/cli"
+)
+
+const (
+	modeDisabled = iota
+	modeCheckMeta
+	modeCheckBlob
+)
+
+const (
+	ModeFlagName      = "scanner.mode"
+	BatchSizeFlagName = "scanner.batch-size"
+	IntervalFlagName  = "scanner.interval"
+	EsRpcFlagName     = "scanner.es-rpc"
+)
+
+type Config struct {
+	Mode      int
+	BatchSize int
+	Interval  int
+	EsRpc     string
+}
+
+func CLIFlags(envPrefix string) []cli.Flag {
+	envPrefix += "_SCANNER"
+	flags := []cli.Flag{
+		cli.IntFlag{
+			Name:   ModeFlagName,
+			Usage:  "Data scan mode, 0: disabled, 1: check meta, 2: check blob",
```

Comment:
can we enable both with different scanner.batch-size and scanner.interval?

Reply:
Do you mean to check both meta and blob? It is unnecessary to check both because the source of the meta is the blob.